### ISSUE TITLE
(Request/suggestion) Option to make fossilized logs harder to harvest/use than regular wood

### ISSUE THREAD

**Issue by Winddbourne** [GlobbyPotato/Rockhounding_Surface#16]
The surface woods are supposed to be fossilized wood. I'd love an option that lets me change the required tool into a pickaxe. Also afterward you should require an axe to turn it into planks. This stuff is fully, or partially, turned to stone . . . I'd love to make that idea more of an immersive part of my pack. :)

**Comment by GlobbyPotato** [comment]
Hi, originally they were made to use pickaxe to cut them, being petrified indeed. but it was reported it conflicted with some automation handling wood, so I had to uniform it. I'll give it a look to see if I can implement some extra config.

**Comment by Winddbourne** [comment]
I've been thinking about this . . . some logs like driftwood probably aren't petrified and should work with automation. Other's are petrified and if I put a rock into my sawmill I'd probably destroy the blade at the very least. 

If it's driftwood you probably shouldn't need anything special to chop it, it's wood, and it should go right into a sawmill . . . but if it's a mineral outcrop that just USED to be wood then it should be like any other rock in the mod, complete with plain blocks and the ability to be carved . . . perhaps even the ability to be carved into "planks" due to being fossil wood and then those planks could be used normally.

Also . . . if you wanted actual woods perhaps there could be a chance of getting saplings if you risk grinding up the fossil wood . . . and those saplings could be used to grow a living tree with leaves and regular wood; thus bringing the species back from the dead.

**Comment by GlobbyPotato** [comment]
yep, driftwood, teredo and the bog woods are actually more withered than petrified.

for crafting I'm keeping it mostly uniformed to the rest, log to planks and then to parts, so players won't have to figure out more than necessary. if you have Chisel installed you have access to the chiseled versions too.

not really want to add saplings, that won't really fit the theme of the mod which is meant to stay realistic. To farm the wood I added a quasi-tech aging system with the Wood Incubator where you simulate the petrifying process. It is made to work alongside the RH Chemistry module but I added alternative recipes to make it work on its own
